### ISSUE TITLE
What is $f_{y_1 y_2}$?

### ISSUE THREAD

**Issue by ZachL1** [EPFL-VILAB/XTConsistency#16]
I am confused about the mapping between different domains, i.e. $f_{y_{1} y_{2}}$. 
In your implementation, is it actually a neural network? (For any two domains $y_1$, $y_2$, have you trained a network $f_{y_1 y_2}$ to map? Then in each step of training, the predicted $y_1'$ and ground truth $y_1$ need to be mapped to $y_2'$ and $y_2$ by $f_{y_1 y_2}$ to calculate the consistency energy and as part of the loss. Is my understanding correct?)
